### PR TITLE
Add donating instructions to the aid package page 

### LINKED ISSUE #325: Display donating instructions on the aid package page
If the user is **not logged in**, the page should look like follows:
Screenshot 2022-08-01 at 22 16 12

The user should be redirected to the https://slredcross.give.asia (open in a new window) when clicking on the donate button.
Text:
Please donate using our secure site: We are currently accepting donations to a general fund that will support multiple aid packages.

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #325 

## Goals

- Add donating instructions to the aid package page 

## Approach

- Added donating instructions

### CODE REVIEW

File: donor-portal/src/pages/package/index.tsx [hunk 1]
```diff
@@ -146,14 +139,33 @@ export default function AidPackageDetailsPage() {
...
-                <button
-                  type="button"
-                  className="btn aid-package-donate-btn"
-                  onClick={handleDonateClick}
-                >
-                  Donate
-                </button>
+                <>
+                  <p className="aid-package-text txt-align">
+                    Please donate using our secure site: We are currently
```
Comment (jayasanka-sack) [inline donor-portal/src/pages/package/index.tsx:146]:
```suggestion
                    Please donate using our secure site. We are currently
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Thanks for the PR @anjula-sack. 

Could you please remove the description header as well?

https://github.com/LSFLK/MedicinesforLK/issues/325#issuecomment-1201469602

**Comment by anjula-sack** [comment]
> Thanks for the PR @anjula-sack.
> 
> Could you please remove the description header as well?
> 
> [#325 (comment)](https://github.com/LSFLK/MedicinesforLK/issues/325#issuecomment-1201469602)

done

File: donor-portal/src/pages/package/index.tsx [hunk 2]
```diff
@@ -146,14 +138,33 @@ export default function AidPackageDetailsPage() {
...
-                <button
-                  type="button"
-                  className="btn aid-package-donate-btn"
-                  onClick={handleDonateClick}
-                >
-                  Donate
-                </button>
+                <>
+                  <p className="aid-package-text txt-align">
+                    Please donate using our secure site, We are currently
```
Comment (jayasanka-sack) [inline donor-portal/src/pages/package/index.tsx:145]:
```suggestion
                    Please donate using our secure site. We are currently
```

**Review by jayasanka-sack — APPROVED** [review]
Thanks @anjula-sack !

**Pull request merged**